src/transformers/__init__.py

```diff
@@ -367,6 +368,13 @@
         "WAVLM_PRETRAINED_CONFIG_ARCHIVE_MAP",
         "WavLMConfig",
     ],
+    "models.x_clip": [
+        "X_CLIP_PRETRAINED_CONFIG_ARCHIVE_MAP",
```

Comment:
For consistency of names, this should be `XCLIP_PRETRAINED_CONFIG_ARCHIVE_MAP` here.